### DISCUSSION TITLE
希望影片2.0

### DISCUSSION THREAD

**Discussion by hite385** [share121/inter-knot#421] [其他]
区区我这种人，竟然能来到这里，不愧是超高校级的幸运，也算是不枉费我这个能力了。 这就是绳网吧，还真是绝望呢。。。不过没关系，我相信大家一定能从绝望中生出更多的希望的！因为绝望永远只会是希望的垫脚石！ 如果想，就杀了我吧，让我成为大家的踏板，我会让大家的希望更加耀眼。 我相信绳匠的大家一定会向我展露更多的希望的！啊哈哈哈哈哈，这是多么美妙啊！  斯巴拉西哟~ 
╯ 　乀 
ヘ　 へ 　
  ′ 
　﹀

**Comment by Kamisato100** [comment]
你也是超高校级的绳匠？

**Comment by hite385** [comment]
我是超高校级的绳斤🤓☝

**Reply by hite385** [reply]
🤔
